app/javascript/dashboard/components/widgets/conversation/ConversationHeader.vue

```diff
@@ -165,14 +167,26 @@ export default {
 </script>
 
 <style lang="scss" scoped>
-.text-truncate {
-  white-space: nowrap;
-  overflow: hidden;
-  text-overflow: ellipsis;
-}
+@import '~dashboard/assets/scss/woot';
 
 .conv-header {
   flex: 0 0 var(--space-jumbo);
+
+  @include breakpoint(640px down) {
+    flex-direction: row;
+  }
+}
+
+.conversation-header--details {
+  display: flex;
+  justify-content: center;
+  flex-direction: row;
+  align-items: center;
+  width: 100%;
+
+  @include breakpoint(640px down) {
```

Comment:
Use pre defined breakpoints or create new ones in foundation settings `@include breakpoint(large up) {`